source/user-manual/reference/centralized-configuration.rst

```diff
@@ -1,17 +1,14 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta::
-  :description: Learn how to remotely configure agents using agent.conf. In this section you will find which capabilities can be configured remotely. 
+.. _reference_shared_conf:
 
```

Comment:
Missing meta description. Please, add this element and make sure it corresponds with the content of the section. 
```suggestion


```